config/webpack/development.js

```diff
@@ -0,0 +1,5 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development'
```

Comment:
at some point I might ask what the actual differences are between environments

Reply:
At this point, no difference at all. But I suppose for production we will want to compress and uglify css as it's 3mb in development ;)